DESCRIPTION

```diff
@@ -52,6 +51,7 @@ Remotes:
     adokter/vol2birdR,
     bioc::rhdf5
 Suggests:
+    aws.s3,
     ggmap (>= 3.0.0),
     knitr,
     rlang,
```

Comment:
Should rlang not be under `Imports` if we are using it to detect if `aws.s3` is installed?